### ISSUE TITLE
[Documentation search] Duplicated search results

### ISSUE THREAD

**Issue by lukas-vlcek** [opensearch-project/documentation-website#6500]
I am experiencing duplicated results in the search results window.

See attached screenshot. The search query is "allocation exclude" (but I am getting for other queries as well) and every single search result is displayed twice. I verified that both such results point to the same URL document.

<img width="1471" alt="Screenshot 2024-02-23 at 14 07 15" src="https://github.com/opensearch-project/documentation-website/assets/205174/a661c5a3-2220-40f6-a976-78f6f0311cc4">

**Comment by hdhalter** [comment]
Thanks, @lukas-vlcek, I'm getting the same thing. @opensearch-project/admin - Can someone please look into this issue with Search? Thanks!

**Comment by bbarani** [comment]
@AMoo-Miki Would you be able to check?

**Comment by hdhalter** [comment]
This looks to be resolved. @lukas-vlcek - is it working now for you?

**Comment by hdhalter** [comment]
This is addressed. If you see it again, @lukas-vlcek, please let us know!

**Comment by lukas-vlcek** [comment]
Unfortunately, I can see it again today.

<img width="1477" alt="Screenshot 2024-04-12 at 15 37 08" src="https://github.com/opensearch-project/documentation-website/assets/205174/63bef669-f7d6-40dc-b5ff-c70f27c57c1e">

Can we reopen the ticket please?

**Comment by hdhalter** [comment]
Yeah, I saw this, too. I'll reopen. We are currently working on search updates, so I'll make sure this is on the list.
